swagger.yml

```diff
@@ -1143,6 +1143,9 @@ paths:
                   items:
                     type: string
                     example: "mail"
+                fixed_last_octet:
+                  type: integer
```

Comment:
is there a concept of nullable int on swagger? if there is we want that

Reply:
openapi 2.0 doesnt.. https://jane.readthedocs.io/en/latest/tips/nullable.html